client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -80,9 +89,13 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 		};
 
 		submitForm = () => {
-			const { fields, site } = this.props;
+			const { fields, settingsFields, site, isJetpack } = this.props;
 			this.props.removeNotice( 'site-settings-save' );
-			this.props.saveSiteSettings( site.ID, fields );
+
+			this.props.saveSiteSettings( site.ID, pick( fields, settingsFields.site ) );
```

Comment:
I don't it's possible. Though, if it's not the case right now, I'm ok with keeping this implementation